nameserver/http.go

```diff
@@ -35,6 +35,11 @@ func (n *Nameserver) HandleHTTP(router *mux.Router, dockerCli *docker.Client) {
 			return
 		}
 
+		if !dns.IsSubDomain(n.domain, hostname) {
+			n.infof("Ignoring name %s, not in %s", hostname, n.domain)
```

Comment:
Did we used to log this previously, i.e. pre-gossip?

If we really *do* want to log, should we provide more context, e.g. the container and ip?